templates/base/scripts/probe_thread_dump.sh

```diff
@@ -79,7 +79,7 @@ function main {
 	then
 		echo -e "${curl_content}"
 
-		local thread_dump=$(jattach $(pidof java) threaddump)
+		local thread_dump=$(jattach $(cat /opt/liferay/liferay.pid) threaddump)
```

Comment:
Maybe the location of the PID could be added as a common var so path (whatever is chosen) isn't duplicated.

Reply:
I will change the code. It will be added as an environment variable, with a default value of /var/run/liferay.pid

Reply:
After some testing, it turns out that the liferay user has no permission to write the pid file there. To make it flexible the new environment variable will expand on the already existing LIFERAY_HOME variable.